docker/hive2-kerberos/Dockerfile

```diff
@@ -85,15 +85,19 @@ RUN chmod 644 /etc/presto/conf/keystore.jks
 RUN keytool -genkey \
     -alias hbase \
     -keyalg RSA \
-    -keysize 1024 \
+    -keysize 2048 \
     -storepass PASS12345 \
     -validity 10000 \
     -keystore /etc/hive/conf/hive.jks \
+    -ext SAN=dns:localhost \
```

Comment:
ok, I have no idea what this is :-D

Reply:
alternative host name in self-signed SSL certificate

Reply:
aaah,  👍